features/device-config/rest/src/main/java/org/opennms/features/deviceconfig/rest/api/DeviceConfigRestService.java

```diff
@@ -126,12 +126,11 @@ Response getDeviceConfigsByInterface(
         @QueryParam("configType") @DefaultValue("") String configType);
 
     /**
-     * Delete a single device config.
-     * @param id
+     * Delete a single or multiple device config.
+     * @param id comma separated string of ids
      */
     @DELETE
-    @Path("{id : \\d+}")
-    void deleteDeviceConfig(@PathParam("id") long id);
+    Response deleteDeviceConfigs(@QueryParam("id") String id);
```

Comment:
A have mixed feelings about removing the single version and therefore break (at least my) assumption on how REST works.

Reply:
didn't get what to do here

Reply:
Keep the version with the single ID and make the one with multiple IDs an additional method.

Reply:
that means two end point?

Reply:
yes

Reply:
Create one more end point which will be used for single id deletion